services/ui/service.cc

```diff
@@ -426,6 +429,12 @@ void Service::OnWillCreateTreeForWindowManager(
 #endif
   } else {
     screen_manager_ = display::ScreenManager::Create();
+#if defined(OS_LINUX) && !defined(OS_CHROMEOS)
+    if (!should_host_viz_) {
```

Comment:
does it make sense to do this in ScreenManager::Create (in services/ui/display/screen_manager_ozone_external.cc )?

Reply:
But how can it access the window server to get a display manager then?

Reply:
change has been removed.